### PR TITLE
[single_node_benchmark] Add TokenV2 Ambasador minting to benchmark

### PR DESCRIPTION
**Pull request by igor-aptos**
modified so that user signs it (sequence number), and owner only provides multisig.

### <samp>🤖 Generated by Copilot at 1b2bdec</samp>

This pull request enhances the transaction generator library and the single node performance test suite to support new transaction types and custom modules. It adds two new transaction types `Batch100Transfer` and `TokenV2AmbassadorMint` for benchmarking batch transfers and custom modules respectively. It also adds support for different kinds of multi-signature transactions and a new custom module `ambassador` for token minting. It updates the address length of the existing custom modules and the expected transactions per second for the new transaction type. It provides the source code of the ambassador module for reference and testing.

### CODE REVIEW

**Review by sitalkedia — APPROVED** [review]
LGTM

**Review by msmouse — APPROVED** [review]
unblock

**Review by msmouse — APPROVED** [review]
unblock

**Review by movekevin — APPROVED** [review]

File: testsuite/module-publish/src/packages/simple/sources/ambassador.move [hunk 1]
```diff
@@ -0,0 +1,220 @@
+module 0xABCD::ambassador {
```
[the remark this answers is not in the document]
Comment (igor-aptos) [reply testsuite/module-publish/src/packages/simple/sources/ambassador.move:1]: I'll check if I can do that on top of #8067, and I'll land as is here.

I do add some additions to the module, but I can do that in the example as well.

**Pull request merged**